optimum/onnxruntime/modeling_seq2seq.py

```diff
@@ -1362,3 +1455,217 @@ def _reorder_cache(past, beam_idx) -> Tuple[Tuple[torch.FloatTensor]]:
     def can_generate(self):
         """Returns True to validate the check that the model using `GenerationMixin.generate()` can indeed generate."""
         return True
+
+
+class ORTModelForPix2Struct(ORTModelForConditionalGeneration, GenerationMixin):
+    """
+    Pix2struct model with a language modeling head for ONNX Runtime inference.
+    """
+
+    # pix2struct cannot be loaded using AutoModel
+    auto_model_class = Pix2StructForConditionalGeneration
+    main_input_name = "flattened_patches"
+
+    # Copied from transformers.models.t5.modeling_t5.T5PreTrainedModel._shift_right with T5->Pix2Struct
+    def _shift_right(self, input_ids):
```

Comment:
@JingyaHuang In transformers, this seem to be used for training. I never really looked at the ONNX config with loss, but does this include a `label` input? i.e., in ORTTrainer, is the export going through e.g. https://github.com/huggingface/transformers/blob/e42587f596181396e1c4b63660abf0c736b10dae/src/transformers/models/t5/modeling_t5.py#L1701-L1703 ?

@krathul I think for now it is reasonable to remove this method. We can have a look at training later.

Reply:
Ok